php_02/php_02.php

```diff
@@ -0,0 +1,87 @@
+<?php
+	$errors = array();
+	$valid = array();
+	
+	if($_POST)
+	{
+		$title = $_POST['title'];
+		if(!$title)
+		{
+			$errors['title'] = 'Името на курса е задължително!';
+			echo $errors['title'];
+		}
+		else if (strlen($title) > 150)
+		{
+			$errors['title'] = 'Името на заглавието трябва до 150 символа!';
+			echo $errors['title'];
+		}
+		else
+		{
+			$valid ['title'] = $title;
+		}
+		
+		$teacher = $_POST ['teacher'];
+		if(!$teacher)
+		{
+			$errors['teacher'] = 'Името на преподавателят е задължително!';
+			echo $errors['teacher'];
+		}
+		else if (strlen($teacher) > 200)
+		{
+			$errors['teacher'] = 'Името на преподавателят трябва до 200 символа!';
+			echo $errors['teacher'];
+		}
+		else
+		{
+			$valid ['teacher'] = $teacher;
+		}
+		
+		$description = $_POST ['description'];
+		if(!$description)
+		{
+			$errors['description'] = 'Описанието е задължително!';
+			echo $errors['description'];
+		}
+		else if (strlen($description) > 200)
+		{
+			$errors['description'] = 'Описанието трябва да е поне 10 символа!';
+			echo $errors['description'];
+		}
+		else
+		{
+			$valid ['description'] = $description;
+		}
+		
+		$group = $_POST ['group'];
+		{
+			$valid ['group'] = $group;
+		}
+		
+		$credits = $_POST ['credits'];
+		{
+			$valid ['credits'] = $credits;
+		}
+		
+		if (count($valid) == 5)
+		{
+			$fileName='data.txt';
```

Comment:
Please provide some output to the user. If he sees a blank page he has no idea what happened and if the action he made was successful.